<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.codelovers.timeset.frontend.HomePage" tests="6" time="13.901" ignored="0" failures="2" timestamp="13 Mar 2017 02:30:27 GMT" skipped="0" hostname="GiangNguyen" errors="0">
  <testcase name="TC2_checkTitleDailySettlement" time="2.055" classname="com.codelovers.timeset.frontend.HomePage"/>
  <testcase name="TC5_checkTitlePosts" time="1.215" classname="com.codelovers.timeset.frontend.HomePage"/>
  <testcase name="TC4_checkTitleBudgetSale" time="2.483" classname="com.codelovers.timeset.frontend.HomePage">
    <failure type="java.lang.AssertionError" message="expected:&lt;昨年対比ランキング&gt; but was:&lt;Frontend&gt;">
      <![CDATA[java.lang.AssertionError: expected:<昨年対比ランキング> but was:<Frontend>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at com.codelovers.timeset.frontend.HomePage.TC4_checkTitleBudgetSale(HomePage.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- TC4_checkTitleBudgetSale -->
  <testcase name="TC6_checkTextAppear" time="1.277" classname="com.codelovers.timeset.frontend.HomePage"/>
  <testcase name="TC3_checkTitleBudgetRanking" time="1.573" classname="com.codelovers.timeset.frontend.HomePage">
    <failure type="java.lang.AssertionError" message="expected:&lt;予算達成ランキング&gt; but was:&lt;予算ランキング&gt;">
      <![CDATA[java.lang.AssertionError: expected:<予算達成ランキング> but was:<予算ランキング>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at com.codelovers.timeset.frontend.HomePage.TC3_checkTitleBudgetRanking(HomePage.java:67)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- TC3_checkTitleBudgetRanking -->
  <testcase name="TC1_checkTitleHomePage" time="5.298" classname="com.codelovers.timeset.frontend.HomePage"/>
</testsuite> <!-- com.codelovers.timeset.frontend.HomePage -->
